index.html

```diff
@@ -47,9 +48,14 @@ <h2 id="project_tagline">A pairing and mentoring group for software engineers (R
 <h3>
 <a id="location" class="anchor" href="#location" aria-hidden="true"><span class="octicon octicon-link"></span></a>Location</h3>
 
-<p>CoverMyMeds<br>
-2 Miranova Pl, Floor 10<br>
-Columbus, OH  </p>
```

Comment:
Sounds good, I was just making a case that both could exist. I just liked the idea of showing the map because it lets the user know exactly where the group meets without actually needing to copy/paste the address into Google Maps.

That and I wanted to toy with Google Maps embeds. :stuck_out_tongue: